### DISCUSSION TITLE
Is it possible to have a thread where people can submit examples or testcases where the text normalizer is not doing perfectly?

### DISCUSSION THREAD

**Discussion by huangruizhe** [openai/whisper#494] [Ideas]
Thanks for the great work!
Just wondered if we can submit or share ad-hoc test cases to make the text normalizer more robust?

For example, some wrong normalizations are:
2020 Third Quarter => 2023rd quarter
Monroe's financial release => monroe is financial release

**Comment by jongwook** [comment]
Yes the normalizer definitely has many rough edges. Your first example seems easy to fix, the second is a bit more tricky because it's hard to differentiate between possessive `'s` and contractions of `is` or `has` with the rule-based approach. But would appreciate more examples like this to be aware of its failure modes!
